packages/file/src/download.coffee.md

```diff
@@ -277,8 +277,7 @@ It would be nice to support alternatives sources such as FTP(S) or SFTP.
         log if match
         then message: "Hash matches as '#{hash_source}'", level: 'INFO', module: 'nikita/lib/file/download' 
         else message: "Hash dont match, source is '#{hash_source}' and target is '#{hash_target}'", level: 'WARN', module: 'nikita/lib/file/download'
-        status = not match
-        unless status
+        unless not match
```

Comment:
How come you are using match now and status before when you didn't change how you initialized match 5 lines above.